packages/launch/src/focused-launch/index.tsx

```diff
@@ -0,0 +1,92 @@
+/**
+ * External dependencies
+ */
+import { Title } from '@automattic/onboarding';
+import { useSelect } from '@wordpress/data';
+import { __ } from '@wordpress/i18n';
+import { TextControl } from '@wordpress/components';
+import * as React from 'react';
+import DomainPicker from '@automattic/domain-picker';
+import { useSite, useDomainSearch } from '../hooks';
+
+import './styles.scss';
+
+function noop( ...args: unknown[] ) {
+	return args;
+}
+
+const FocusedLaunch: React.FunctionComponent = () => {
+	const { getSite } = useSelect( ( select ) => select( 'core' ) ) as any;
+	const title = getSite()?.title;
+
+	const site = useSite();
+
+	const [ siteTitle, setSiteTitle ] = React.useState( title );
+	const [ siteDomainName ] = React.useState( site.currentDomainName );
+	const domainSearch = useDomainSearch();
+
+	return (
+		<div className="focused-launch__container">
+			<div className="focused-launch__section">
+				<Title>{ __( "You're almost there", 'launch' ) }</Title>
+				<p className="focused-launch__caption">
+					{ __(
+						'Prepare for launch! Confirm a few final things before you take it live.',
+						'launch'
+					) }
+				</p>
+			</div>
+			<div className="focused-launch__step">
+				<div className="focused-launch__data-input">
+					<div className="focused-launch__section">
+						<TextControl
+							className="focused-launch__input"
+							label={ __( '1. Name your site', 'launch' ) }
```

Comment:
For i18n to work I think we need to follow the pattern from `domain-picker` and `plans-grid` and replace `'launch'` with `'__i18n_text_domain__'` everywhere in `launch` package. 
See https://github.com/Automattic/wp-calypso/pull/46557 for details.

Reply:
What value show `__i18n_text_domain__` be replaced with, within the `@automattic/launch` package?

The domain picker, for example, uses `default`

Reply:
The translation domain has been changed from `'launch'` to `__i18n_text_domain__`